src/kadmin/ktutil/ktutil.c

```diff
@@ -149,6 +150,7 @@ void ktutil_add_entry(argc, argv)
         }
         if ((strlen(argv[i]) == 2) && !strncmp(argv[i], "-k", 2)) {
             kvno = (krb5_kvno) atoi(argv[++i]);
+            use_kvno++;
```

Comment:
Oh, I didn't realize this was unintentional. kvno 0 is useful for creating keytabs when you don't know the principal's current kvno and don't have a way to look it up. The version of krb5-workstation that yum installs on Rhel 7 has krb5-kvno-230379.patch, and the ability to specify kvno 0 is convenient. The kvno does not seem to be included in the AS-REP.